client/app/pages/groups/GroupsList.jsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { react2angular } from 'react2angular';
+
+import Button from 'antd/lib/button';
+import Tooltip from 'antd/lib/tooltip';
+import { Paginator } from '@/components/Paginator';
+
+import { wrap as liveItemsList, ControllerType } from '@/components/items-list/ItemsList';
+import { ResourceItemsSource } from '@/components/items-list/classes/ItemsSource';
+import { StateStorage } from '@/components/items-list/classes/StateStorage';
+
+import LoadingState from '@/components/items-list/components/LoadingState';
+import EmptyState from '@/components/items-list/components/EmptyState';
+import ItemsTable, { Columns } from '@/components/items-list/components/ItemsTable';
+
+import CreateGroupDialog from '@/components/groups/CreateGroupDialog';
+import DeleteGroupButton from '@/components/groups/DeleteGroupButton';
+
+import { Group } from '@/services/group';
+import settingsMenu from '@/services/settingsMenu';
+import { currentUser } from '@/services/auth';
+import navigateTo from '@/services/navigateTo';
+import { routesToAngularRoutes } from '@/lib/utils';
+
+class GroupsList extends React.Component {
+  static propTypes = {
+    controller: ControllerType.isRequired,
+  };
+
+  listColumns = [
+    Columns.custom((text, group) => (
+      <div>
+        <a href={'groups/' + group.id}>{group.name}</a>
+        {(group.type === 'builtin') && <span className="label label-default m-l-10">built-in</span>}
+      </div>
+    ), {
+      field: 'name',
+      width: null,
+    }),
+    Columns.custom((text, group) => (
+      <Button.Group>
+        <Button href={`groups/${group.id}`} onClick={e => e.stopPropagation()}>Members</Button>
+        {currentUser.isAdmin && (
+          <Button href={`groups/${group.id}/data_sources`} onClick={e => e.stopPropagation()}>Data Sources</Button>
+        )}
+      </Button.Group>
+    ), {
+      width: '1%',
+      className: 'text-nowrap',
+    }),
+    Columns.custom((text, group) => {
+      const canRemove = group.type !== 'builtin';
+      const button = (
+        <DeleteGroupButton
+          className="w-100"
+          disabled={!canRemove}
+          group={group}
+          onClick={() => this.onGroupDeleted()}
+        >
+          Delete
+        </DeleteGroupButton>
+      );
+      return canRemove ? button : (
+        <Tooltip placement="top" title="Cannot delete built-in group" mouseLeaveDelay={0}>{button}</Tooltip>
```

Comment:
Disabled buttons don't seem to capture `mouseLeave` events very well :confused:. The only way to close this tooltip is unhovering the tooltip itself. Perhaps wrap the button with something else (like a `<div>`) fixes it?

![tooltip-disabled-button](https://user-images.githubusercontent.com/3356951/53239905-f45b1d00-367b-11e9-9b60-224a1327c6f4.gif)

Reply:
@gabrieldutra Thanks! It wasn't working because `Tooltip` expects a `Button` as direct child, but I created a custom component. Fixed now (just wait for build)